cli/cmd/data/tables_queries.yml

```diff
@@ -10,5 +10,51 @@
       - cloudquery/sqlite:
           query: |
             sqlite3 %q SELECT account_id, instance_id, region, launch_time FROM aws_ec2_instances ORDER BY launch_time ASC LIMIT 10
+          args:
+            - connection_string
+- aws_ec2_ebs_volumes:
+    description: |
+      Run the following command to get your unattached EBS volumes:
+    destinations:
+      - cloudquery/postgresql:
+          query: |
+            SELECT * FROM aws_ec2_ebs_volumes WHERE jsonb_array_length(attachments) = 0
+      - cloudquery/sqlite:
+          query: |
+            sqlite3 %q SELECT * FROM aws_ec2_ebs_volumes WHERE json_array_length(attachments) = 0
+          args:
+            - connection_string
+- aws_iam_users:
+    description: |
+      Run the following command to get IAM users without MFA:
+    destinations:
+      - cloudquery/postgresql:
+          query: |
+            select user_name, arn, create_date, user_id, tags from aws_iam_users AS u WHERE u.user_name NOT IN (SELECT user_name FROM aws_iam_mfa_devices)
+      - cloudquery/sqlite:
+          query: |
+            sqlite3 %q select user_name, arn, create_date, user_id, tags from aws_iam_users AS u WHERE u.user_name NOT IN (SELECT user_name FROM aws_iam_mfa_devices);
```

Comment:
This will work only if you sync `aws_iam_mfa_devices` as well, so maybe we need to update the logic to include dependencies